### ISSUE TITLE
provide test results

### ISSUE THREAD

**Issue by chaoyan1073** [brucechou1983/CheXNet-Keras#1]
Thanks for you providing your implementation! Could you please upload your best test results？ So I can compare it with my running result.

**Comment by brucechou1983** [comment]
Since the original paper does not provide the detail of HOW they choose the dev/test sets, comparing those numbers is kind of meaningless. I don't want users of this project to be misled so I intentionally do not provide my "best" test results.

In case you want to make sure your code is executed correctly, normally you should be able to get avg. AUROC from ~ 0.81 to 0.82 if you just randomly split the dataset like the baseline in the source code. I will release a carefully selected dev/test sets in the near future, and I could share the results based on that. But again, I will not compare the results with the original paper since the dev/test sets could be quite different.

**Comment by chaoyan1073** [comment]
Hi Bruce, thanks for your clarifications. I just want to know if I am running the model correctly. Thanks again.

**Comment by taewookim** [comment]
@brucechou1983  so this is comparable to the [pytorch version of chexnet](https://github.com/arnoweng/CheXNet)?

**Comment by brucechou1983** [comment]
@taewookim No, I don't compare with pytorch implementation since it has its own dataset splitting strategy. Please read my comment above. Thanks.

**Comment by chaoyan1073** [comment]
I do not think that data splitting strategy matters since the training and testing lists have been provided. The Pytorch version is supposed to split the data according to given training and testing lists, though I have not checked that.

**Comment by brucechou1983** [comment]
@chaoyan1073 For sure you can run this project with lists in the pytorch implementation. I won't do that because the way they split the data means nothing to me. Last time I check their lists, I could not understand their design of dev/test sets, so I won't do that comparison. I would only release my splitting strategy with my own reason and also the performance. If you have any insight on the way they handle datasets, please share your thoughts.

**Comment by chaoyan1073** [comment]
I think I did not make myself clear. What I try to convey is that those people who released the [Chest Xray-14](https://nihcc.app.box.com/v/ChestXray-NIHCC)  data have provided the training and testing lists along with the data. People are supposed to split the data according to the "official" training and testing lists.

**Comment by brucechou1983** [comment]
Got your point. Could you confirm that Pytorch implementation use it? If so, how do they select dev set that has the same distribution with test set?

**Comment by billhhh** [comment]
@brucechou1983 Hi, Bruce, did you also implement Wang's algorithm (the author of Chest X ray dataset), or did you have any idea about it?

**Comment by brucechou1983** [comment]
@billhhh Nope. PR is welcome :)
